reference/dic_tags.rst

```diff
@@ -470,6 +471,77 @@ To add a new rendering strategy - in addition to the core strategies like
 :class:`Symfony\\Component\\HttpKernel\\Fragment\\FragmentRendererInterface`,
 register it as a service, then tag it with ``kernel.fragment_renderer``.
 
+kernel.locale_aware
+-------------------
+
+.. versionadded:: 4.3
```

Comment:
You need to target v4.4 and here swap to 4.4 :)

Reply:
The merge branch is 4.4, but it's still OK to say that this was introduced in 4.3.